Maple2.Model/Metadata/Server_Table/InstancedFieldTable.cs

```diff
@@ -0,0 +1,6 @@
+using System.Collections.Generic;
```

Comment:
nit picky but can we make the directory be named without the _ ?
so ServerTable instead